lib/dfe_sign_in_api.rb

```diff
@@ -0,0 +1,35 @@
+module DFESignIn
+  class API
+    BASE_URL = DFE_SIGN_IN_URL
+
+    attr_accessor :dfe_sign_in_password
```

Comment:
What was the thinking behind making this an `attr_accessor`?

Reply:
Changed this to an instance variable in 935caaf